packages/hooks-react/src/useProtectedMedia.ts

```diff
@@ -7,7 +7,9 @@ import useContentProtection from './useContentProtection';
 
 export default function useProtectedMedia(item: PlaylistItem) {
   const apiService = getModule(ApiService);
-  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) => apiService.getMediaById(item.mediaid, token, drmPolicyId));
+  const contentProtectionQuery = useContentProtection('media', item.mediaid, (token, drmPolicyId) =>
+    apiService.getMediaById(item.mediaid, undefined, token, drmPolicyId),
```

Comment:
Can we make the language the last argument of the function? That way the signature remains the same.